automatic/dropbox/tools/chocolateyinstall.ps1

```diff
@@ -1,9 +1,11 @@
 ﻿$ErrorActionPreference  = 'Stop'
+ if(!$PSScriptRoot){ $PSScriptRoot = Split-Path $MyInvocation.MyCommand.Path -Parent }
 . "$PSScriptRoot\helper.ps1"
 
 $version = '72.3.127-beta'
 
 if (!(IsVersionAlreadyInstalled $version)) {
+    if ( $version -match "beta" ) { $version = $version -replace("-beta","") }
```

Comment:
Why did you add this line?
It isn't used anywhere else, other than in the version installed check.

Reply:
The version check is trying to check the version installed which dropbox doesn't install with a `-beta` tag. I can make a gist of what I'm saying because I have the `--pre` version installed. The current check is a `False` and will install the dropbox version again.
Mainly it is a fix that should go in the `helper.ps1` file.

Reply:
I'll make the change in the update file so a valid version is used for the check.

Reply:
If that was the intention, then you added the replacement check in the wrong place, it would have needed to be added before the version check takes place, not after.

Anyways, it is better to just pass the actual version during update than doing such replacements during install.